Playbooks/ThinkstCanary-Alert-Ingestion/azuredeploy.json

```diff
@@ -0,0 +1,95 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
```

Comment:
Kindly add metadata of playbook : 
Consider below link as a reference

https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-BlockIP-URL/azuredeploy.json

Reply:
Added metadata section to playbook with relevant details as requested.